src/Monarch/VirtualDom/NS.purs

```diff
@@ -0,0 +1,6 @@
+module Monarch.VirtualDom.NS where
+
+foreign import kind NS
```

Comment:
Where is NS coming from?

Reply:
It's PureScript's strange syntax for defining new kinds! Unfortunately PureScript doesn't have DataKinds yet.